runtimes/spiritnet/src/lib.rs

```diff
@@ -1018,10 +1056,58 @@ pub type Executive = frame_executive::Executive<
 	Runtime,
 	// Executes pallet hooks in the order of definition in construct_runtime
 	AllPalletsWithSystem,
-	// EthereumMigration<Runtime>,
-	parachain_staking::migration::StakingPayoutRefactor<Runtime>,
+	(
+		// "Bound uses of call" <https://github.com/paritytech/polkadot/pull/5729>
+		pallet_preimage::migration::v1::Migration<Runtime>,
+		pallet_scheduler::migration::v3::MigrateToV4<Runtime>,
+		pallet_democracy::migrations::v1::Migration<Runtime>,
+		runtime_common::migrations::AddCTypeBlockNumber<Runtime>,
+		// The migration above must be run as last since it checks that all pallets are using the new StorageVersion
+		// properly.
+		runtime_common::migrations::MigrateToNewStorageVersion<Runtime>,
+	),
```

Comment:
we might be missing some migrations here as discussed on slack, this comment is just as a reminder